testem.js

```diff
@@ -1,12 +1,19 @@
 /* eslint-env node */
 module.exports = {
-  "test_page": "tests/index.html?hidepassed",
-  "disable_watching": true,
-  "launch_in_ci": [
-    "PhantomJS"
+  test_page: 'tests/index.html?hidepassed',
+  disable_watching: true,
+  launch_in_ci: [
+    'Chrome'
   ],
-  "launch_in_dev": [
-    "PhantomJS",
-    "Chrome"
-  ]
+  launch_in_dev: [
+    'Chrome'
```

Comment:
no more Phantom - 2.15 defaults to headless Chrome for all tests. See the [release blog post](https://emberjs.com/blog/2017/09/01/ember-2-15-released.html#toc_changes-in-ember-cli-2-15) for deets